api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const x = req.get('X-Access-Token');
```

Comment:
`token` would be a better variable name than `x`, it is best to avoid one letter variable names in most cases.